### PR TITLE
Added package.json.

### PR DESCRIPTION
**Pull request by mathieumg**
Following discussion in #68.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "bootstrap-social",
+  "version": "4.9.0",
```
Comment (lipis) [inline package.json:3]: is it possible to remove the version so there will be no need to maintain that?

Comment (mathieumg) [inline package.json:3]: Sadly, I don't think that can be done: https://docs.npmjs.com/files/package.json#version

Are you only git tagging when releasing at the moment? If so, there is the `npm version` command that updates the `package.json` with the new version and git tags automatically at once: https://docs.npmjs.com/cli/version

**Comment by mathieumg** [comment]
Before you merge anything, let me just add the `repo` field in the package, it will facilitate people's way to GitHub if they have issues.

**Comment by dbkaplun** [comment]
Can we get this merged please? This repo needs a `package.json` anyway; future revisions can go into separate PRs.

**Comment by mathieumg** [comment]
> Can we get this merged please? This repo needs a package.json anyway; future revisions can go into separate PRs.

:+1:

**Comment by mathieumg** [comment]
:tada: Will you publish to https://www.npmjs.com/package/bootstrap-social now that you own it? Thanks!

**Comment by lipis** [comment]
not sure how though :)

**Comment by mathieumg** [comment]
https://docs.npmjs.com/getting-started/publishing-npm-packages

Basically, if you are already logged in to npm and in the directory of your project, `npm publish` will suffice :+1:

**Comment by mathieumg** [comment]
Now published, thank you very much! :cherries:

**Comment by lipis** [comment]
:) Any idea on how to update the readme?

**Comment by mathieumg** [comment]
It uses the current README.md when you publish.

**Pull request merged**